src/MOI_wrapper.jl

```diff
@@ -431,5 +433,139 @@ end
+conflict is not purged, and any calls to the above attributes will return values
+for the original conflict without a warning.
+"""
+function compute_conflict(model::Optimizer)
+    model.conflict = c_api_getconflict(model.inner)
+    return
+end
+
+function _ensure_conflict_computed(model::Optimizer)
+    if model.conflict === nothing
+        error("Cannot access conflict status. Call `CPLEX.compute_conflict(model)` first. " * 
+              "In case the model is modified, the computed conflict will not be purged.")
+    end
+end
+
+"""
+    ConflictStatus()
+
+Return an `MOI.TerminationStatusCode` indicating the status of the last computed conflict.
+
+If a minimal conflict is found, it will return `MOI.OPTIMAL`. If the problem is feasible, it will
+return `MOI.INFEASIBLE`. If `compute_conflict` has not been called yet, it will return 
+`MOI.OPTIMIZE_NOT_CALLED`. 
+"""
+struct ConflictStatus <: MOI.AbstractModelAttribute  end
+MOI.is_set_by_optimize(::ConflictStatus) = true
 
+function MOI.get(model::Optimizer, ::ConflictStatus) 
+    if model.conflict === nothing
+        return MOI.OPTIMIZE_NOT_CALLED
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_MINIMAL
+        return MOI.OPTIMAL
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_FEASIBLE
+        return MOI.INFEASIBLE
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_CONTRADICTION
+        return MOI.OTHER_LIMIT
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_DETTIME_LIM
+        return MOI.TIME_LIMIT
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_IT_LIM
+        return MOI.ITERATION_LIMIT
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_MEM_LIM
+        return MOI.MEMORY_LIMIT
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_NODE_LIM
+        return MOI.NODE_LIMIT
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_OBJ_LIM
+        return MOI.OBJECTIVE_LIMIT
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_TIME_LIM
+        return MOI.TIME_LIMIT
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_USER
+        return MOI.OTHER_LIMIT
+    else
+        return MOI.OTHER_LIMIT
+    end
+end
+
+function MOI.supports(::Optimizer, ::ConflictStatus)
+    return true
+end
+
+"""
+    ConstraintConflictStatus()
+
+A Boolean constraint attribute indicating whether the constraint participates in the last computed conflict.
+"""
+struct ConstraintConflictStatus <: MOI.AbstractConstraintAttribute end
+MOI.is_set_by_optimize(::ConstraintConflictStatus) = true
+
+function MOI.get(model::Optimizer, ::ConstraintConflictStatus, index::MOI.ConstraintIndex{<:MOI.SingleVariable, <:LQOI.LE}) 
+    _ensure_conflict_computed(model)
+    var_in_conflict = findfirst(isequal(LQOI.get_column(model, model[index]) - 1), model.conflict.colind)
+    return model.conflict.colstat[var_in_conflict] == CPLEX.CPX_CONFLICT_MEMBER || 
+        model.conflict.colstat[var_in_conflict] == CPLEX.CPX_CONFLICT_UB
```

Comment:
What are the other possible values for the conflict?

Reply:
There are six possible values, depending on whether CPLEX could prove a bound is within the conflict or not: 

 * CPLEX.CPX_CONFLICT_MEMBER: both bounds in the conflict
 * CPLEX.CPX_CONFLICT_UB: upper bound in the conflict
 * CPLEX.CPX_CONFLICT_LB: lower bound in the conflict
 * CPLEX.CPX_CONFLICT_POSSIBLE_MEMBER: both bounds POSSIBLY in the conflict
 * CPLEX.CPX_CONFLICT_POSSIBLE_UB: POSSIBLY upper bound in the conflict
 * CPLEX.CPX_CONFLICT_POSSIBLE_LB: POSSIBLY lower bound in the conflict